src/runner.rs

```diff
@@ -224,4 +226,29 @@ When your transaction is complete, enter 'commit' or 'abort' as appropriate."#,
         }
         Ok(())
     }
+
+    pub(crate) async fn handle_status(&self) -> Result<()> {
+        let region = self.deps.env.region().value.name().to_string();
+        let request_url = &format!("https://session.qldb.{}.amazonaws.com/ping", region);
```

Comment:
I don't think this is right.

I think we should be:

1. respecting the value of `env.qldb_session_endpoint()` if set
2. else, using `Region`to build the endpoint directly (as opposed to string interpolation)

I'm not sure how to go about (2), so it's OK if you open an issue to fix later, but I do think (1) should be addressed in this PR.

Reply:
I updated the logic to check for `qldb_session_endpoint` first. I wasn't able to access the `qldb_session_endpoint()` method  in `environment.rs` because it was private so I made it public. 

For your second point, are you asking to build the URL using [Url crates](https://docs.rs/url/1.7.2/url/struct.Url.html)?